src/Explorer/Explorer.ts

```diff
@@ -194,6 +194,7 @@ export default class Explorer {
 
   // features
   public isGalleryPublishEnabled: ko.Computed<boolean>;
+  public isCodeOfConductEnabled: ko.Computed<boolean>;

```

Comment:
If possible, I would scope this to gallery publish: e.g `isPublishCodeOfConductEnabled`. So it's more self-explanatory as to which feature this applies to.

Reply:
this code of conduct needs to be accepted for both publish and for viewing the public gallery (since download and usage of public data is possible in that case)

Reply:
Ok then maybe do `isGalleryCodeOfConductEnabled`. Otherwise it would appear it's the CodeOfConduct for DataExplorer as a whole.

Reply:
will leave it as such for now, since this is a feature flag that will be removed shortly.